src/styles/index.css

```diff
@@ -254,10 +254,28 @@ a {
 	line-height: 1.5;
 	font-weight: 400;
 
-	/* color-scheme: light dark; */
-
 	font-synthesis: none;
 	text-rendering: optimizeLegibility;
 	-webkit-font-smoothing: antialiased;
 	-moz-osx-font-smoothing: grayscale;
 }
+
+input[type='file']::file-selector-button {
+	background-color: #ff7647;
+	background: url('https://raw.githubusercontent.com/dscnitrourkela/project-udon/060792975fadc77fa8e10eb2e351e3ca70c66d41/src/assets/images/Upload-button.png')
+		no-repeat;
+	cursor: pointer;
+	color: rgba(0, 0, 0, 0);
+	border: none;
+	width: 110px;
+	padding: 0.2em;
+	margin: 0vw 0.3vw;
+	transition: 1s;
+
+	@media only screen and (max-width: 970px) {
+		width: 60px;
+	}
+	@media only screen and (max-width: 768px) {
+		width: 110px;
+	}
+}
```

Comment:
Is there any need for putting this in global CSS if it is only being used in one page.

Reply:
moved css in form folder
https://github.com/dscnitrourkela/project-udon/pull/11/commits/dab1d403fd68012efb4e7332df564830bead177d